package/gluon-radvd/files/etc/init.d/gluon-radvd

```diff
@@ -4,12 +4,12 @@ USE_PROCD=1
 START=50
 
 start_service() {
-  [ -x /lib/gluon/radvd/arguments ] && {
-    procd_open_instance
-    procd_set_param command /usr/sbin/uradvd $(/lib/gluon/radvd/arguments)
-    procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
-    procd_set_param netdev br-client
-    procd_set_param stderr 1
-    procd_close_instance
-  }
+	[ -x /lib/gluon/radvd/arguments ] &&
```

Comment:
I think it would be even nicer to change this into

    [ -x /lib/gluon/radvd/arguments ] || return 1